thewalrus/quantum/gaussian_checks.py

```diff
@@ -125,24 +130,27 @@ def fidelity(mu1, cov1, mu2, cov2, hbar=2, rtol=1e-05, atol=1e-08):
     if not n0 == n1 == m0 == m1 == l0 == l1:
         raise ValueError("The inputs have incompatible shapes")
 
-    v1 = cov1 / hbar
-    v2 = cov2 / hbar
-    deltar = (mu1 - mu2) / np.sqrt(hbar / 2)
-    n = n0 // 2
-    W = sympmat(n)
-
-    si12 = np.linalg.inv(v1 + v2)
-    vaux = W.T @ si12 @ (0.25 * W + v2 @ W @ v1)
-    p1 = vaux @ W
-    p1 = p1 @ p1
-    p1 = np.identity(2 * n) + 0.25 * np.linalg.inv(p1)
-    if np.allclose(p1, 0, rtol=rtol, atol=atol):
-        p1 = np.zeros_like(p1)
+    # We first convert all the inputs to quantities where hbar = 1
+    sigma1 = cov1 / hbar
+    sigma2 = cov2 / hbar
+    deltar = (mu1 - mu2) / np.sqrt(hbar)
+
+    Omega = sympmat(n0 // 2)  # The symplectic matrix
```

Comment:
I would recommend using lowercase for these variables, since capitalization would be used for classes, and thus get highlighted differently. Any reason for changing to capitalized versions?

Reply:
I knew this would come up! So I am just trying to follow the paper and LaTeX naming, i.e., Omega is meant to be $\Omega$. The Walrus has no classes.

Reply:
But if you feel strongly about this I am happy to change it.

Reply:
Sorry about this. It's a case of programming vs. academic conventions. I would say it's still better to keep lowercase, and follow Python syntax rules here. Either having it as `omega` or even `big_omega` (but I think the former is probably better).